src/equations/acoustic_perturbation_2d.jl

```diff
@@ -66,7 +64,7 @@ function AcousticPerturbationEquations2D(; v_mean_global::NTuple{2,<:Real}, c_me
 end
 
 
-varnames(::typeof(cons2cons), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_prime",
+varnames(::typeof(cons2cons), ::AcousticPerturbationEquations2D) = ("v1_prime", "v2_prime", "p_tilde",
```

Comment:
While reading this, I was wondering a few things:
* What is a good name for this new pressure-like quantity `p/c_bar^2`? Is it a "reduced pressure" or a "scaled pressure"? Do you know of any naming precedence which we could follow?
* Does it feel weird that we have one variable that seems to be not a perturbed value? That is, wouldn't the reduced pressure (just calling it this way for now) better be called something like `p_tilde_prime`?
* Should we make the reduced pressure truly a non-dimensional pressure by adding `1/rho` to the mix? However, it would also feel weird to have just one non-dimensional quantity and the rest dimensional. OTOH, `c_bar` and `rho_bar` are not "proper" non-dimensionalization quantities, since they are only constant in time but not in space, making it impossible to extract them, e.g., from the divergence operator.

I am leaning towards calling this pressure `p_tilde_prime` - to emphasize that while it's not a real pressure (but rather something with the units of a density) it is still a perturbed quantity - and leave the rest of the PR as it is.

@jlchan @andrewwinters5000 @gregorgassner @ranocha It would be great to get some thoughts from you on this as well :-)

Reply:
- I like "scaled pressure", but I don't know of a naming precedent. 
- I think it's fine to have a non-perturbed value as a solution variable so long as it's documented.
- No opinion on `1/rho` - I'm not very good at non-dimensionalization, so I don't think I would notice. 

I might have some other thoughts after looking at the docs. Is there a way to preview them for this PR?

Reply:
- Having no in-depth experience with these equations, `p_tilde_prime` is pretty ... obfuscated? Something like scaled pressure gives me at least a better hint but YMMV.
- The other decisions are fine, I think.

Reply:
> Is there a way to preview them for this PR?

I don't think so since it's from a fork. I we built the docs from forks, we had a security problem.

Reply:
If it doesn't complicate the formulation (e.g., adding more source terms) I support including `1/rho` in the non-dimensionalization for `p_tilde`.

Reply:
Thanks for all the feedback so far!

> If it doesn't complicate the formulation (e.g., adding more source terms) I support including `1/rho` in the non-dimensionalization for `p_tilde`.

Unfortunately, it does just this. It will leave a `rho_bar` term in front of the time derivative of the pressure equation, which cannot be removed (as far as I can tell) without introducing a new source term.

Regarding naming, would the following make sense to you:
* Conservative: `u_prime`, `v_prime`, `p_prime_scaled`
* Primitivative: `u_prime`, `v_prime`, `p_prime`

?

Reply:
> Unfortunately, it does just this. It will leave a `rho_bar` term in front of the time derivative of the pressure equation, which cannot be removed (as far as I can tell) without introducing a new source term.

Doesn't seem to be worth the effort, does it?

> Regarding naming, would the following make sense to you:
> * Conservative: `u_prime`, `v_prime`, `p_prime_scaled`
> * Primitivative: `u_prime`, `v_prime`, `p_prime`

Yes

Reply:
> Regarding naming, would the following make sense to you:
> 
>     * Conservative: `u_prime`, `v_prime`, `p_prime_scaled`
> 
>     * Primitivative: `u_prime`, `v_prime`, `p_prime`

Yes, these variable names are good from my side

Reply:
Thanks again for the feedback!

@lchristm Could you please change the variable name from `p_tilde` to `p_prime_scaled`? Regarding the Latex notation in the docstring, I think you could do something like `\tilde{p}'` to signify that it both perturbed (the `'`) and scaled (the `~`). If you have another, better idea, please feel encouraed to share it!

Once finished, please re-request a review from just me.